config/r4e/v1_1_exp/translate_test.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2022 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.)
+
+package v1_1_exp
+
+import (
+	"fmt"
+	"testing"
+
+	base "github.com/coreos/butane/base/v0_5_exp"
+	"github.com/coreos/butane/config/common"
+	"github.com/coreos/ignition/v2/config/util"
+	"github.com/coreos/vcontext/path"
+	"github.com/coreos/vcontext/report"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test that we error on unsupported fields for r4e
+func TestTranslateInvalid(t *testing.T) {
+	type InvalidEntry struct {
+		Kind report.EntryKind
+		Err  error
+		Path path.ContextPath
+	}
+	tests := []struct {
+		In      Config
+		Entries []InvalidEntry
+	}{
+		{
```

Comment:
It might make sense to add some high level comments around each config for glance value. 

i.e. https://github.com/coreos/butane/blob/main/config/fcos/v1_4/translate_test.go#L56

Reply:
There is a comment on the test function explaining what the test is about: `// Test that we error on unsupported fields for r4e` 
Should I repeat something similar to that on each config?

Reply:
I see! I agree that it would make it easier to jump into the code and understand the test. I will update it with comments :+1: